content/en/docs/concepts/networking/function-access.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Function Access"
+linkTitle: "Function Access"
+weight: 3440
+description:
+---
+
+> This documentation will assume you are using default OpenFunction [Gateway](https://openfunction.dev/docs/concepts/networking/gateway/#default-gateway) and you have a synchronous function named `function-sample`.
+
+## Access Function within the cluster
+### Access via Function internal address
+OpenFunction will create this service for every synchronous `Function`: `{{.Name}}.{{.Namespace}}.svc.cluster.local`. This service will be used to provide the Function internal address.
+
+Get `Function` internal address by running following command:
+```shell
+export FUNC_INTERNAL_ADDRESS=$(kubectl get function function-sample -o=jsonpath='{.status.addresses[?(@.type=="Internal")].value}')
+```
+
+This address provides the default method for accessing functions within the cluster, it's suitable for use as `sink.url` of `EventSource`.
+
+Access `Function` using curl in pod:
+```shell=
+kubectl run --rm ofn-test -i --tty --image=radial/busyboxplus:curl -- curl -sv $FUNC_INTERNAL_ADDRESS
+```
+
+### Access via Function external address
+> The Function external address is usually used for external access, but you can also use it within the cluster for development or testing.
+You should configure local domain first, see [Configure Local Domain](https://openfunction.dev/docs/concepts/networking/local-domain).
+
+Get `Function` external address by running following command:
+```shell
+export FUNC_EXTERNAL_ADDRESS=$(kubectl get function function-sample -o=jsonpath='{.status.addresses[?(@.type=="External")].value}')
+```
+
+Access `Function` using curl in pod:
+```shell=
+kubectl run --rm ofn-test -i --tty --image=radial/busyboxplus:curl -- curl -sv $FUNC_EXTERNAL_ADDRESS
```

Comment:
We needn't to use a pod to access external address ?

Reply:
needs, Config Local Domain only configures the DNS in the cluster, access on the node directly may needs to use dnsmasq.